config/initializers/sidekiq.rb

```diff
@@ -0,0 +1,7 @@
+Sidekiq.configure_server do |config|
+  config.redis = { url: Settings.redis.sidekiq.uri, size: 15 }
```

Comment:
see http://manuelvanrijn.nl/sidekiq-heroku-redis-calc/

Reply:
ah that link is great!

Reply:
yes, I was quite happy to find it :)